docs/developer/release-process.md

```diff
@@ -244,12 +244,18 @@ Once the PR has been created, have a look at it (eg. remove any development chan
 
 > Some issues can arise here, so please check the app versions are being properly updated at once and ensure you have the latest changes in the PR branch. Note that the [bitnami-bot](https://github.com/bitnami-bot) usually performs some automated commits to the main branch that might collide with the changes in our PR. In particular, it will release a new version of the chart with the updated images.
 
-## 5 - Publish the GitHub release
+## 5 - Check Dockerfiles and notify the proper teams
+
+Eventually, as the Kubeapps code continues to evolve, some changes are often introduced in our own [development container images](https://hub.docker.com/u/kubeapps). However, those changes won't get released in the official Bitnami repository unless we manually notify the proper team to also include those changes in their building system.
+
+> As part of this release process, each Kubeapps component's Dockerfile _must_ compared against the one in the previous release. If they functionally differ each other, the Bitnami Content team _must_ be notified.
```

Comment:
Thanks Antonio. Can we even compare them, or is this something we need to ask the Bitnami content team to do? I tried yesterday to find the Dockerfile, but only found a Dockerfile which pulls a tar.gz and expands it... so quite opaque.

Reply:
We can compare them with the internal `gitlist` tool, but yep, we wouldn't find a `Dockerfile` as is, but a kind of "recipe" with their way to build the image. Then we would have to figure out if these recipe steps match our Dockerfile; so it is really a manual process likely to be performed by the proper team.